frontend-react/src/pages/Upload.tsx

```diff
@@ -19,6 +21,57 @@ import {
 import { showError } from "../components/AlertNotifications";
 import Spinner from "../components/Spinner";
 
+const TransitionBanner = () => {
+    return (
+        <Alert
+            type="info"
+            heading="CVS uploader will be moving to SimpleReport"
```

Comment:
typo - "CVS" should read "CSV"

Reply:
Ohhhh, we're _not_ sending these to CVS? 😂